pkg/system/phase2_creating.go

```diff
@@ -594,6 +594,11 @@ func (r *Reconciler) ReconcileBackingStoreCredentials() error {
 	if util.IsAWSPlatform() {
 		return r.ReconcileAWSCredentials()
 	}
+
+	if util.IsHCIPlatform() {
+		// return r.ReconcileAzureCredentials()
```

Comment:
remove this comment. Why do we need this part? just for this print? debugging?

Reply:
Done. Removed.